.pre-commit-config.yaml

```diff
@@ -68,6 +68,8 @@ repos:
   rev: v13.0.1
   hooks:
   - id: clang-format
+    types: [file]
+    files: '.*\.(c|cpp|cxx|h|hpp|tpp)(\.in)?$'
```

Comment:
This does not yet seem to work :thinking:

Reply:
Once this works, I to also need to add `.json` here as we use this for json formatting as well #1222

Reply:
Ah, could be python regex: https://pre-commit.com/#regular-expressions